src/styles/core/_settings.scss

```diff
@@ -104,7 +104,7 @@ $line-height: nth(map-get($type,0), 2) !default;
 
 //baseline
 $color: tn-color(text) !default;
-$background-color: tn-color(inverse) !default;
+$background-color: tn-color(text-inverse) !default;
```

Comment:
Using the text-inverse color group not for text. Should be using the neutrals color group.